script/lib/util.py

```diff
@@ -83,6 +84,7 @@ def download(text, url, path):
     if hasattr(ssl, '_create_unverified_context'):
       ssl._create_default_https_context = ssl._create_unverified_context
 
+    print url
```

Comment:
Is it left intentionally?

Reply:
Yes. This was a useful tracer when I was hitting 404s and I thought it would be similarly useful if our CI ever dies due to fetch failure

Reply:
Can we add some more context to the log then, `print 'Downloading: "url" to "path"'`